enos/modules/get_local_metadata/scripts/version.sh

```diff
@@ -5,5 +5,6 @@
 set -eu -o pipefail
 
 pushd "$(git rev-parse --show-toplevel)" > /dev/null
-make ci-get-version
+CWD="$(pwd)"
```

Comment:
Won't this expect the `get-local-version.sh` script to exist relative to the root of the repository?

This script failed when executing it locally. First because `/usr/bin/env bash` as a shebang didn't work on my mac.

I did get it to work by changing it to